src/styledConstants.ts

```diff
@@ -2,5 +2,58 @@ import styled from "styled-components";
 
 export const colors = {
   violet: "#8657DD",
```

Comment:
Designers asked to do more colors categories like in their library (you can see more in my commit)